src/components/Table/GridTable.stories.tsx

```diff
@@ -2144,3 +2146,111 @@ export function MinColumnWidths() {
     </div>
   );
 }
+
+enum EditableRowStatus {
+  Foo = "Foo",
+  Bar = "Bar",
+}
+
+type EditableRowData = {
+  kind: "data";
+  id: string;
+  data: { id: string; name: string; status: EditableRowStatus; value: number; date?: Date };
+};
+type EditableRow = EditableRowData | HeaderRow;
+
+export function HighlightFields() {
+  const [rows, setRows] = useState<GridDataRow<EditableRow>[]>([
+    simpleHeader,
+    {
+      kind: "data" as const,
+      id: "1",
+      data: { id: "1", name: "Tony Stark", status: EditableRowStatus.Foo, value: 1, date: jan1 },
+    },
+    {
+      kind: "data" as const,
+      id: "2",
+      data: { id: "2", name: "Natasha Romanova", status: EditableRowStatus.Foo, value: 2, date: jan2 },
+    },
+    {
+      kind: "data" as const,
+      id: "3",
+      data: { id: "3", name: "Thor Odinson", status: EditableRowStatus.Bar, value: 3, date: jan29 },
+    },
+  ]);
+
+  const setRow = useCallback((rowId: string, field: keyof EditableRowData["data"], value: any) => {
+    setRows((rows) =>
+      rows.map((row) =>
+        row.kind === "data" && row.id === rowId ? { ...row, data: { ...row.data, [field]: value } } : row,
+      ),
+    );
+  }, []);
+
+  const nameColumn: GridColumn<EditableRow> = {
+    header: "Name",
+    data: ({ name }) => name,
+  };
+
+  const selectColumn: GridColumn<EditableRow> = {
+    header: "Status",
+    data: (row) => ({
+      content: (
+        <SelectField
+          label=""
+          options={Object.values(EditableRowStatus).map((status) => ({ label: status, code: status }))}
+          value={row.status}
+          onSelect={(status) => setRow(row.id, "status", status)}
+        />
+      ),
+    }),
+    w: "120px",
+  };
+
+  const date1Column: GridColumn<EditableRow> = {
+    header: "Date",
+    data: (row) => ({
+      content: (
+        <DateField
+          label=""
+          value={row.date}
+          onChange={(date) => setRow(row.id, "date", date)}
+          hideCalendarIcon
+          format="medium"
+        />
+      ),
+    }),
+    w: "120px",
+  };
+
+  const date2Column: GridColumn<EditableRow> = {
+    header: "Date",
+    data: (row) => ({
+      content: (
+        <DateField
+          label=""
+          value={row.date}
+          onChange={(date) => setRow(row.id, "date", date)}
+          hideCalendarIcon
+          format="medium"
+        />
+      ),
+    }),
+    w: "120px",
+  };
+
+  const style = getTableStyles({ bordered: true, allWhite: true, highlightOnHover: true });
+
+  return (
+    <div css={Css.m2.$}>
+      <GridTable
+        columns={[nameColumn, selectColumn, date1Column, date2Column]}
+        rows={rows}
+        style={{
+          ...style,
+          rowHoverColor: Palette.Blue50,
```

Comment:
I think this is already the default `rowHoverColor` now? From the TableStyles change on line 183?